core/pom.xml

```diff
@@ -38,6 +38,21 @@
         <scope>test</scope>
     </dependency>
 
+      <dependency>
+        <groupId>com.vlkan.hrrs</groupId>
+        <artifactId>hrrs-servlet-filter-base64</artifactId>
+        <version>0.5</version>
+      </dependency> 
+      <dependency>
+        <groupId>com.google.guava</groupId>
+        <artifactId>guava</artifactId>
+        <version>23.6-jre</version>
```

Comment:
we are currently importing guava 19.0 in the root pom.xml ... perhaps we should upgrade that rather than load two separate versions of guava